cabal-install/tests/UnitTests/Distribution/Client/ProjectConfig.hs

```diff
@@ -730,7 +730,7 @@ instance Arbitrary CompilerFlavor where
         --TODO: [code cleanup] export knownCompilerFlavors from D.Compiler
         -- it's already defined there, just need it exported.
         knownCompilerFlavors =
-          [GHC, GHCJS, NHC, YHC, Hugs, HBC, Helium, JHC, UHC]
+          [GHC, GHCJS, NHC, YHC, Hugs, HBC, Helium]
```

Comment:
Can you please fix the TODO here while you're at it?

Reply:
I intentionally did not to avoid mixing up the "remove JHC" change with an "expand the Cabal API" change.

I'm assuming that adding the knownCompilerFlavors will implicitly expand the Cabal API... is that assumption right, and do we want that?

If so, it'll happily do a folllow-up for that.

(I suppose I could add a Haddock note about the API only being for testing, or something?)